src/helm/benchmark/server.py

```diff
@@ -28,6 +33,27 @@ def serve_config():
         )
 
 
+# Shim for running helm-server for old suites from old version of helm-summarize
+# that do not contain schema.json
+# TODO(2023-05-01): Remove this.
```

Comment:
What's this date?

Reply:
Arbitrarily chosen - we will remove this in a few months after most users have moved to the new version of helm-summarize.